lib/cjson/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+set(CJSON_MODULE_DIR ${ZEPHYR_BASE}/../modules/lib/cjson)
```

Comment:
I'm surprised to see it done this way. I was expecting to see the build system files in the module itself, e.g. with a zephyr/module.yml. cc @tejlmand

Reply:
yes, the correct way would be to use `${ZEPHYR_CJSON_MODULE_DIR}` in code below.

That of course requires that cjson is made into a proper Zephyr module.
In which case this file should be moved to: `modules/cjson/CMakeLists.txt`.

The same goes for `lib/cjson/Kconfig`.